pytrends/request.py

```diff
@@ -85,12 +101,21 @@ def GetGoogleCookie(self):
                 else:
                     proxy = ''
                 try:
-                    return dict(filter(lambda i: i[0] == 'NID', requests.get(
-                        f'{BASE_TRENDS_URL}/?geo={self.hl[-2:]}',
-                        timeout=self.timeout,
-                        proxies=proxy,
-                        **self.requests_args
-                    ).cookies.items()))
+                    if self.impersonate:
+                        return dict(filter(lambda i: i[0] == 'NID', curl_requests.get(
```

Comment:
Change from requests.get to requests.post.

Reply:
The only weird thing is that the curl_requests.get() works fine in my tests, but the requests.get() doesn't.

Reply:
Oops yeah I commented on the wrong line. I agree, keep the `get` for curl.

Reply:
Oh oops, commented on the wrong line. Yeah ignore this one.